### ISSUE TITLE
[f-1] Audit and update all Chat UI's and invite notifications 

### ISSUE THREAD

**Issue by NickTooker** [Expensify/App#38498]
### High level problem:
Currently, our invite notifications don’t really follow a cohesive pattern. There are many ways to invite people and then even more emails/notifications that are sent as a result of adding/updating users to workspaces, UCRs, invoice rooms and so on. We need to do an audit of all that messaging and update/standardize it all so:
- The messages look great and contains 0 bugs
- The text of the messages is updated and standardized. 
- We make these messages look closer to product notifications than to our current plain text.

**Comment by maddylewis** [comment]
Alright, was able to "clear my plate" so to speak on a few things, so i will review the thread you posted last week, nick, and then start updating our tracking sheet.

**Comment by maddylewis** [comment]
moving to daily to focus on this week

**Comment by maddylewis** [comment]
next steps here - https://expensify.slack.com/archives/C01TF865XTL/p1711379820860659

**Comment by NickTooker** [comment]
<img width="935" alt="Screenshot 2024-03-25 at 1 14 36 PM" src="https://github.com/Expensify/App/assets/33639890/5a4cdf5a-3e8e-48ad-b13d-e08dd393067d">
I am getting no invite at all when adding myself via mention to a new room.

**Comment by NickTooker** [comment]
Started a [DOC](https://docs.google.com/document/d/1u6-aMZBTAe3GUJRKx0sBg6RnGM6ALFFL1zy4aNwGVJA/edit) to keep track of screenshots for styling in later phases!

**Comment by maddylewis** [comment]
- Nick + Marketing Team QA'ing to gather the varying messages
- Once the QA phase is complete, Nick will call me back in to help with the copy and links and such!

**Comment by NickTooker** [comment]
Planning out a pre-design to decide what needs to be QA'ed.

**Comment by NickTooker** [comment]
@danielrvidal and I are still working through finalizing the pre-design that will go out this week.

**Comment by danielrvidal** [comment]
@NickTooker can you update this one when you get a chance.

**Comment by NickTooker** [comment]
Earnings and a few launches this week are taking priority but I will be writing up a recap to the pre-design tomorrow and we will move forward with the HL design doc at the end of the week.

**Comment by NickTooker** [comment]
### [LATEST DISCUSSION HERE.](https://expensify.slack.com/archives/C036QM0SLJK/p1714767525501349)

**Comment by NickTooker** [comment]
**[LATEST DISCUSSION HERE.](https://expensify.slack.com/archives/C036QM0SLJK/p1714767525501349)**

**Comment by NickTooker** [comment]
Working through the roomDescription Design Doc first before returning to this initiative.

**Comment by NickTooker** [comment]
The latest update is that we have pivoted from our original goal through [this discussion.](https://expensify.slack.com/archives/C036QM0SLJK/p1717782944026509?thread_ts=1714767525.501349&cid=C036QM0SLJK) 

First, we'd like to fill out [THIS DOCUMENT](https://docs.google.com/document/d/1E7Hw0H71K0til4GYpPfiuwh5cbREsAMVmPVjBJOBUyk/edit) to get all of our Chat UI components and rules agreed upon. From there, we can move forward with a UI update, and then this GH which pertains to the notifications that the UI update will inform.

**Comment by NickTooker** [comment]
No change since my last update above, we are still working through the best next steps in the thread referenced, along with the rules and UI/email components within the document I've referenced.

**Comment by NickTooker** [comment]
@danielrvidal is going to lead up this initiative while I am OOO:

[Here is the thread we are working from. ](https://expensify.slack.com/archives/C036QM0SLJK/p1714767525501349)
[Here is the GH where we originally asked @shawnborton to create mock-ups for roomDescriptions. ](https://github.com/Expensify/Expensify/issues/396584)
[Here is the design doc for updating roomDescriptions that we can transform into the all encompassing design doc for UI of Chat/Notifications.](https://docs.google.com/document/d/1FlgV-3jx21Y53jDcXm_jEQZ09QVgBscwfcmmwdZYOdQ/edit)
[Here is the Components and Rules document that David created outlining how Chat UI / Notifications ideally work. ](https://docs.google.com/document/d/1E7Hw0H71K0til4GYpPfiuwh5cbREsAMVmPVjBJOBUyk/edit?pli=1#heading=h.rcji4nvzrjhr)

**Comment by danielrvidal** [comment]
Still working away on this. I've been prioritizing some Stage3/CVP/GBTA related things so this will have to come next week.

**Comment by danielrvidal** [comment]
Still working on this.

**Comment by danielrvidal** [comment]
Still on hold.

**Comment by danielrvidal** [comment]
We're now a/b testing the updated version: https://github.com/Expensify/App/issues/47108

**Comment by danielrvidal** [comment]
Still working on the A/B test. We're just waiting on the F1 room to open.

**Comment by danielrvidal** [comment]
Still waiting on a/b testing linked above. So this will be on hold. Moving to monthly to keep it open but no update needed for a while.

**Comment by trjExpensify** [comment]
You moving this into f1 project, Dan?

**Comment by danielrvidal** [comment]
Yea, I just updated to that project and will track it there.

**Comment by danielrvidal** [comment]
We're a/b testing the subject lines to start.

**Comment by danielrvidal** [comment]
This is still ongoing but not high priority.

**Comment by danielrvidal** [comment]
Still open but no progress. We're prioritizing a/b testing.

**Comment by NickTooker** [comment]
This is still on hold pending A/B testing.

**Comment by NickTooker** [comment]
I am going to close this until we have concluded AB testing - let me know if you disagree.
